.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,31 @@
+name: Cleanup
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # run every day at 10 server time (UTC)
+
+  workflow_dispatch:
+
+jobs:
+  report:
+    name: "Clean up old results"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Allure
```

Comment:
Do we need this step here?

Reply:
Yes, we need Allure in case some reports need to be recreated.